src/app/overview/overview.component.ts

```diff
@@ -473,10 +483,15 @@ export class OverviewComponent extends SubscriptionManager implements OnInit {
     }
   }
 
-  formatNLV(prefix: string, name: string, valid: boolean): NameLabelValid {
+  formatNLV(
```

Comment:
Please avoid acronym here, since `NLV` is not even clear what is supposed to represent

Reply:
Renamed